src/redux/redux-store.js

```diff
@@ -0,0 +1,6 @@
+import * as Redux from "redux";
```

Comment:
Don't import whole module. Instead of your app have to use only things that really will be used.

For this use case, use named imports

Reply:
```suggestion
import { createStore } from 'redux'
```